pkgs/top-level/perl-packages.nix

```diff
@@ -915,11 +915,11 @@ let
   };
 
   AppSqitch = buildPerlModule {
-    version = "1.1.0";
+    version = "1.3.1";
     pname = "App-Sqitch";
     src = fetchurl {
-      url = "mirror://cpan/authors/id/D/DW/DWHEELER/App-Sqitch-v1.1.0.tar.gz";
-      hash = "sha256-7hRs111jAIN+bKVZuwveJH1CEjyWssXUsoAPONPj0as=";
+      url = "mirror://cpan/authors/id/D/DW/DWHEELER/App-Sqitch-v1.3.1.tar.gz";
```

Comment:
Minor comment, but it seems common for Perl packages to not interpolate `${version}` in the URLs and I don't really know why. A handful do it though so maybe it's just a holdover from old-style

Reply:
Yeah, I think that pattern in `perl-packages.nix` isn't so good. So, I'm trying to spread some improvement. But, if it's not recommended for some reason that I didn't consider then I'm happy to edit.

Reply:
Hm, I actually wasn't doing what I said/thought I was. I pushed a commit to recycle `pname` and `version` based on this comment.